src/ert/gui/model/snapshot.py

```diff
@@ -103,7 +103,6 @@ def prerender(
         # If there are no realizations, there's nothing to prerender.
         if not snapshot.data().get(ids.REALS):
             return None
```

Comment:
`snapshot.data().get(ids.REALS)` is always truthy in the tests. Is this case not needed?

Reply:
it's impossible to know, in the real world (without reading and understanding like LOTS of code)